packages/core/src/CSL/cslToCore/cslToCore.ts

```diff
@@ -141,16 +140,16 @@ export const txBody = (body: CSL.TransactionBody): Cardano.NewTxBodyAlonzo => {
   };
 };
 
-export const txBootstrap = (bootstraps?: CSL.BootstrapWitnesses): BootstrapWitness[] | undefined => {
+export const txBootstrap = (bootstraps?: CSL.BootstrapWitnesses): Cardano.BootstrapWitness[] | undefined => {
   if (!bootstraps) return;
-  const result: BootstrapWitness[] = [];
+  const result: Cardano.BootstrapWitness[] = [];
   for (let i = 0; i < bootstraps.len(); i++) {
     const bootstrap = bootstraps.get(i);
     result.push({
       addressAttributes: bootstrap.attributes().toString(),
       chainCode: bootstrap.chain_code().toString(),
-      key: bootstrap.vkey.toString(),
-      signature: bootstrap.signature.toString()
+      key: typedHex(bootstrap.vkey.toString()),
+      signature: typedHex(bootstrap.signature.toString())
```

Comment:
```suggestion
      signature: Cardano.Ed25519Signature(bootstrap.signature.toString())
```

Reply:
Done in 96f35a2cd2c61a2e4295b77d7757d2a68bb9c4cf